labman/db/process.py

```diff
@@ -3514,7 +3526,8 @@ def generate_prep_information(self):
                         for k, v in df.orig_name.iteritems()]
             # If the orig_name is none (bc this item is a control),
             # use its content
```

Comment:
Please update the comment ... as I understand it, this is no longer referring to "orig_name" but now to "orig_name2"

Reply:
Will do!